src/assets/AssetsContractController.ts

```diff
@@ -126,7 +126,7 @@ export class AssetsContractController extends BaseController<
    * @param address - ERC20 asset contract address.
    * @returns Promise resolving to the 'decimals'.
    */
-  async getERC20TokenDecimals(address: string): Promise<string> {
+  async getERC20TokenDecimals(address: string): Promise<BN> {
```

Comment:
This public method still returns `BN`. I am guessing this always returned `BN` in practice, but the type was wrong?